client/client.go

```diff
@@ -30,7 +30,8 @@ import (
 
 //nolint gosec
 
-const defaultBaseURL = "https://metrics-collector.cloudability.com"
+const DefaultBaseURL = "https://metrics-collector.cloudability.com"
+const EUBaseURL string = "https://metrics-collector-eu.cloudability.com"
 const defaultTimeout = 1 * time.Minute
 const defaultMaxRetries = 5
```

Comment:
Consistency? no `string` and first character lowercase?

Reply:
I did not mean to drop the string type declaration, I will add it back.

The reason to capitalize d -> D is we now use the variable outside of the client package so it needs to be a public var.